src/components/common/FormPage/PasswordChecks.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
```

Comment:
I don't think this needs to be a component

Reply:
It needs to, it'll make the file too long and hard to navigate

Reply:
Is this the password strength indicator?

Reply:
No, it's the part that shows the users the if the password check the test and the red and green bars

Reply:
It's a component in that section/part/component.

Reply:
Rough pseudocode. Less than 5 lines

```suggestion
------- JS --------
let calcWidth = 100 -  (100 / passwordError.length())
------- HTML --------
<div> // container: Takes 100% width
    <div style={{width: `${calcWidth}%`}}></div> // Progress div
</div>
```

Reply:
Except it's three different bars and not one that shows progress.

Reply:
So basically, a password strength indicator

Reply:
Hmmm, my bad, looking at the design. It works fine, carry on. I will resolve the comment

Reply:
Okay, yes, that's the password strength indicator, but your suggestion won't work (read my comment above yours). Every time I fetch upstream and rebase, it brings back files I've deleted and reverts my work to what it was two commits ago, even if I clearly chose "accept current change" when resolving conflicts.

I have exams tomorrow, engineering maths, my laptop's dead, I've been replying from my phone and I'm tired. I'm not changing anything else, not another git fetch upstream or git rebase upstream/dev. We'll meet in stage 5.